src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/HenleggBehandling/HenleggBehandling.tsx

```diff
@@ -37,12 +37,11 @@ interface HenleggÅrsakSelect extends HTMLSelectElement {
 const StyledVeivalgTekst = styled(Normaltekst)`
     position: relative;
     top: -32px;
-`;
-
-const StyledVeivalgIkon = styled(Oppfylt)`
-    position: relative;
-    top: 7px;
-    margin-right: 10px;
+    svg {
```

Comment:
Måtte flytte dette opp hit for å få med stylingen, ettersom det ellers ville være `styled(styled(Element))` hvor ytterste `styled()` ikke blir fulgt.